docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -9,52 +9,95 @@ priority: 2
 * Table of Contents
 {:toc}
 
-This guide describes how to run [Presto](https://prestodb.io/) with Alluxio, so
-that you can easily use Presto to query Hive tables stored in Alluxio's tiered storage.
+## Overview
+
+[Presto](https://prestodb.io/)
+is an open source distributed SQL query engine for running interactive analytic queries
+on data at a large scale.
+This guide describes how to run Presto to query Alluxio as a co-located distributed cache for
+Hive tables, where the data source can be AWS S3, Azure blob store, HDFS and many others.
+As a result, as long as a block is cached on one Alluxio worker node,
```

Comment:
Unclear what "As a result" is referring to, perhaps just remove it

Reply:
rephrased